src/Nncase.Importer/HuggingFace/HuggingFaceUtils.cs

```diff
@@ -0,0 +1,335 @@
+﻿// Copyright (c) Canaan Inc. All rights reserved.
+// Licensed under the Apache license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading;
+using DryIoc.ImTools;
+using NetFabric.Hyperlinq;
+using Newtonsoft.Json.Linq;
```

Comment:
`System.Text.Json` & `Newtonsoft.Json.Linq` both are used?

Reply:
`System.Text.Json`  used in safetensor load
`Newtonsoft.Json.Linq` used in config.json load